packages/koa-shopify-webhooks/src/register.ts

```diff
@@ -1,27 +1,39 @@
 import {Method, Header} from '@shopify/network';
 import {WebhookHeader, Topic} from './types';
 
+export enum ApiVersion {
+  April19 = '2019-04',
+  July19 = '2019-07',
+  Unstable = 'unstable',
+  Unversioned = 'unversioned',
+}
+
 export interface Options {
   topic: Topic;
   address: string;
   shop: string;
   accessToken: string;
+  apiVersion: ApiVersion;
```

Comment:
Since I rudely didn't respond to this in Slack... yes, I think this is breaking.  Unless we default to the old behaviour.  

My preference would be to bite the bullet here because -- in my limited understanding -- it makes sense to always specify a version.

Reply:
I totally agree. Partners have been having issues with the old behaviour and certain topics anyway.